File: android-app/src/main/java/fr/cph/chicago/entity/dto/DivvyDTO.java
package fr.cph.chicago.entity.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

import fr.cph.chicago.entity.BikeStation;
import lombok.Data;


@Data
public class DivvyDTO {

    @JsonProperty("executionTime")
    private String executionTime;
    @JsonProperty("stationBeanList")
    private List<BikeStation> stations;

    public DivvyDTO() {
    }
}

File: android-app/src/main/java/fr/cph/chicago/entity/dto/DivvyDTO.kt
package fr.cph.chicago.entity.dto

import com.fasterxml.jackson.annotation.JsonProperty
import fr.cph.chicago.entity.BikeStation


class DivvyDTO {

    @JsonProperty("executionTime")
    val executionTime: String? = null
    @JsonProperty("stationBeanList")
    val stations: List<BikeStation>? = null
}
